demo/src/test/java/amazon/SearchFieldTest.java

```diff
@@ -0,0 +1,137 @@
+package amazon;
+
+import java.time.Duration;
+import java.util.List;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class SearchFieldTest {
+   protected WebDriver driver;
+   protected String searchData = "laptop"; //default
+
+   public SearchFieldTest() {    
+      // before before
+      System.setProperty("webdriver.chrome.driver", "src/test/resources/webdriver/chromedriver.exe");
+      this.driver = new ChromeDriver();
+      driver.manage().window().maximize();
+      driver.manage().timeouts().pageLoadTimeout(Duration.ofSeconds(10));
+      driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+   }
+
+   @BeforeMethod
+   public void before(){
+      driver.get("https://www.amazon.com/");
+      PageFactory.initElements(driver, this);
+   }
+
+   @AfterTest
+   public void afterTest() {
+      driver.close();
+      driver.quit();
+   }
+
+   @FindBy(id = "twotabsearchtextbox")
```

Comment:
Using @FindBy annotations was expected in the next practice. It is not relevant to have page fields in a test class. Hope to see the PageObject pattern in the next task :)

Reply:
ok